components/Carousel/Carousel.js

```diff
@@ -1,8 +1,48 @@
 class Carousel {
+    constructor(photoCarousel) {
+        this.photoCarousel = photoCarousel;
 
+        this.leftButton = document.querySelector('.left-button');
+        this.rightButton = document.querySelector('right-button');
+        this.photoData = this.photoCarousel.dataset.tab;
+
+        this.photo = document.querySelectorAll(`.image[data-tab='${this.photoData}']`);
+
```

Comment:
I recommend having a `this.currentIndex` in your constructor here that begins at zero that you can manipulate using your `leftClick` and `rightClick` methods, it might make this easier to get working :)